gtfs-realtime/spec/en/reference.md

```diff
@@ -381,7 +379,8 @@ A selector for an entity in a GTFS feed. The values of the fields should corresp
 | **agency_id** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | At least one specifier must be given - all fields in an EntitySelector cannot be empty.   
 | **route_id** | [string](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | At least one specifier must be given - all fields in an EntitySelector cannot be empty.
 | **route_type** | [int32](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | At least one specifier must be given - all fields in an EntitySelector cannot be empty.
-| **trip** | [TripDescriptor](#message-tripdescriptor) | Conditionally required | One | At least one specifier must be given - all fields in an EntitySelector cannot be empty.
+| **direction_id** | [uint32](https://developers.google.com/protocol-buffers/docs/proto#scalar) | Conditionally required | One | At least one specifier must be given - all fields in an EntitySelector cannot be empty.
```

Comment:
The new fields need better descriptions - you can probably just copy the text directly from `TripDescriptor`, just make sure it's clear that it's referring to a trip.